AppController/djinn.rb

```diff
@@ -3925,27 +3928,48 @@ def restore_from_db?
     @options['restore_from_tar'] || @options['restore_from_ebs']
   end
 
+
   def build_taskqueue()
     Djinn.log_info('Building uncommitted taskqueue changes')
     extras = TaskQueue::OPTIONAL_FEATURES.join(',')
     if system('pip install --upgrade --no-deps ' +
               "#{APPSCALE_HOME}/AppTaskQueue[#{extras}] > /dev/null 2>&1")
-      Djinn.log_info('Finished building taskqueue')
-    else
-      Djinn.log_error('Unable to build taskqueue')
+      if system('pip install ' +
```

Comment:
Unusual alignment.

I would avoid nesting when convenient. For example, you could do
```
unless [first cmd], log_error + return
unless [second cmd], log_error + return
log_success
```
It would also allow you to differentiate between errors.